### DISCUSSION TITLE
Spiral Login Help

### DISCUSSION THREAD

**Discussion by aurelVZ** [SpiralLinux/SpiralLinux-project#305] [Q&A]
Hello

I put Spiral on USB using Unebotin

it is version LXQT i found on sourceForge and restart PC
anfter booting i get Login window 

OTHER:

**But i cannot find anywhere what is  username & password**

any help?

**Comment by geckolinux** [comment]
Hello, by default there is automatic login with no password. Please use one of the supported methods to write it to USB:
https://github.com/SpiralLinux/SpiralLinux-project/wiki#in-linux
